data/people/martin-joiner.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Martin Joiner",
+  "description": "",
+  "meetup-id": "",
```

Comment:
Not sure what to do about an empty meetup ID - left it as an empty string for now although normally I think they're supposed to be integers...

Reply:
This is his meetup ID: https://www.meetup.com/php-sw/members/181882602/

Also MeetupID is not mandatory for a Person (I know in the past we've had external speakers come in who've not been on meeup.

Reply:
Ah cool, I couldn't find him when I was searching. Thanks - updated now